include/ocpp/common/websocket/websocket_libwebsockets.hpp

```diff
@@ -72,7 +72,8 @@ class WebsocketTlsTPM final : public WebsocketBase {
     void on_writable();
 
     /// \brief Called when a message is received over the TLS websocket, calls the message callback
-    void on_message(void* msg, size_t len);
+    void on_message(std::string&& message);
```

Comment:
can be const

Reply:
const with rvalue references does not make sense, unless you try to disable it.